modules/local/star/align/main.nf

```diff
@@ -1,48 +1,61 @@
-// Import generic module functions
```

Comment:
Is this and the other modules with a meta.yml file taking from nf-core? If yes, in the long term it is preferable to use the modules from the nf-core repository to make the pipeline maintenance easier. Also, is there functionality to patch the modules, so even if you need to have small rearrangements still you can use the original module, see [here](https://nf-co.re/tools/#create-a-patch-file-for-a-module).

Reply:
Oh this is nice feature, I missed it. I've made a patch for STAR.